### PR TITLE
ci: coverage report: allow ignoring files by path, ignore maelstrom

### PR DESCRIPTION
**Pull request by nrainer-materialize**
### Motivation

Ignore maelstrom files from coverage analysis.

### Test
https://buildkite.com/materialize/coverage/builds/252 which contains this branch + [danhhz:persist_txn_operator](https://github.com/danhhz/materialize/tree/persist_txn_operator)

### CODE REVIEW

File: misc/python/materialize/cli/ci_coverage_pr_report.py [hunk 1]
```diff
@@ -85,10 +93,20 @@ def find_modified_lines() -> Coverage:
                 start = int(parts)
                 length = 1
             for line_nr in range(start, start + length):
-                coverage[file][line_nr] = None
+                coverage[file_path][line_nr] = None
     return coverage
 
 
+def ignore_file_in_coverage_report(file_path: str) -> bool:
+    if file_path.endswith(".rs"):
```
Comment (philip-stoev) [inline misc/python/materialize/cli/ci_coverage_pr_report.py:101]: `if not` ?

**Review by philip-stoev — CHANGES_REQUESTED** [review]
I think the conditional in `ignore_file...` is wrong.

File: misc/python/materialize/cli/ci_coverage_pr_report.py (hunk 1 above)
Comment (nrainer-materialize) [reply misc/python/materialize/cli/ci_coverage_pr_report.py:101]: Thank you! 🙈
This is what happens when you change the condition in the last second before the commit (changing `include_file...` to `ignore_file...`)

Comment (nrainer-materialize) [reply misc/python/materialize/cli/ci_coverage_pr_report.py]: ✅ Fixed.

**Review by philip-stoev — APPROVED** [review]

**Review by def- — APPROVED** [review]

**Comment by nrainer-materialize** [comment]
Successfully verified on https://buildkite.com/materialize/coverage/builds/255.

**Pull request merged**